HW02/Lizaveta_Muromskaya/spec/my_array_methods_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative 'my_array_methods'
+using MyArrayMethods
+
+RSpec.describe 'MyArrayMethods' do
+  empty_array = []
+  array = [1, 2, 3, 4, 5]
```

Comment:
Can you use `let` instead of these variables?
```suggestion
  let(:empty_array) { [] }
  let(:array) { [1, 2, 3, 4, 5] }
```